src/qibo/backends/__init__.py

```diff
@@ -8,7 +8,7 @@
 from qibo.config import log, raise_error
 
 
-def construct_backend(backend, platform=None, runcard=None):
+def construct_backend(backend, platform=None, runcard=None, token=None, provider=None):
```

Comment:
Maybe we should consider using 
```suggestion
def construct_backend(backend, **kwargs):
```

What do you think @scarrazza?